### ISSUE TITLE
Tracking WebView is broken in iOS 17.4.x

### ISSUE THREAD

**Issue by Ajaxy** [scenee/FloatingPanel#629]
I am using Capacitor View Controller to display WebView within Floating Panel.

```Swift
let capVc = CAPBridgeViewControllerForBottomSheet()

let fpc = FloatingPanelController()
fpc.set(contentViewController: capVc)
fpc.track(scrollView: capVc!.webView!.scrollView)
```

It all works fine, however, with the recent release of iOS 17.4.x the last line stopped working. It looks like `WebView.scrollView` behavior changed and FloatingPanelController can not properly track it anymore.

Did anyone face a similar problem or have a possible solution?

**Comment by scenee** [comment]
Thank you for your report. I've confirmed this problem is occurring in sample app of the library. And then I found the gestures of the WKWebView has changed a bit. I'm trying to fix this problem now.

**Comment by scenee** [comment]
I fixed this issue #630 and merged it to master branch. I will release it in v2.8.3 soon. Thanks.

**Comment by Ajaxy** [comment]
Thanks @scenee for your great project.

**Comment by Ajaxy** [comment]
@scenee Do you have an idea when the update will come out? Otherwise, is there a way to install it from master?

**Comment by scenee** [comment]
I'm sorry for the delay, but I've released v2.8.3 now. Thank you for your help.

**Comment by Ajaxy** [comment]
Thank you!
